{"text": "2021-06-23 18:52:21.167 | ERROR    | app.utils:log_error:32 - TypeError(\"'age' is an invalid keyword argument for User\")\n", "record": {"elapsed": {"repr": "0:00:34.323943", "seconds": 34.323943}, "exception": null, "extra": {"user": "someone"}, "file": {"name": "utils.py", "path": "E:\\Pycharm\\learn-fast-api\\app\\utils.py"}, "function": "log_error", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 32, "message": "TypeError(\"'age' is an invalid keyword argument for User\")", "module": "utils", "name": "app.utils", "process": {"id": 19948, "name": "SpawnProcess-1"}, "thread": {"id": 22712, "name": "MainThread"}, "time": {"repr": "2021-06-23 18:52:21.167004+06:00", "timestamp": 1624452741.167004}}}
{"text": "2021-06-23 18:53:22.464 | ERROR    | app.utils:log_error:32 - TypeError(\"'age' is an invalid keyword argument for User\")\n", "record": {"elapsed": {"repr": "0:01:35.621859", "seconds": 95.621859}, "exception": null, "extra": {"user": "someone"}, "file": {"name": "utils.py", "path": "E:\\Pycharm\\learn-fast-api\\app\\utils.py"}, "function": "log_error", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 32, "message": "TypeError(\"'age' is an invalid keyword argument for User\")", "module": "utils", "name": "app.utils", "process": {"id": 19948, "name": "SpawnProcess-1"}, "thread": {"id": 22712, "name": "MainThread"}, "time": {"repr": "2021-06-23 18:53:22.464920+06:00", "timestamp": 1624452802.46492}}}
{"text": "2021-06-23 18:57:58.951 | ERROR    | app.utils:log_error:32 - TypeError(\"'age' is an invalid keyword argument for User\")\n", "record": {"elapsed": {"repr": "0:00:10.428986", "seconds": 10.428986}, "exception": null, "extra": {"user": "someone"}, "file": {"name": "utils.py", "path": "E:\\Pycharm\\learn-fast-api\\app\\utils.py"}, "function": "log_error", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 32, "message": "TypeError(\"'age' is an invalid keyword argument for User\")", "module": "utils", "name": "app.utils", "process": {"id": 16548, "name": "SpawnProcess-5"}, "thread": {"id": 21252, "name": "MainThread"}, "time": {"repr": "2021-06-23 18:57:58.951817+06:00", "timestamp": 1624453078.951817}}}
{"text": "2021-06-23 18:59:30.145 | ERROR    | app.utils:log_error:32 - TypeError(\"'age' is an invalid keyword argument for User\")\n", "record": {"elapsed": {"repr": "0:00:09.458269", "seconds": 9.458269}, "exception": null, "extra": {"user": "someone"}, "file": {"name": "utils.py", "path": "E:\\Pycharm\\learn-fast-api\\app\\utils.py"}, "function": "log_error", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 32, "message": "TypeError(\"'age' is an invalid keyword argument for User\")", "module": "utils", "name": "app.utils", "process": {"id": 22424, "name": "SpawnProcess-6"}, "thread": {"id": 9248, "name": "MainThread"}, "time": {"repr": "2021-06-23 18:59:30.145525+06:00", "timestamp": 1624453170.145525}}}
{"text": "2021-06-23 23:28:27.855 | ERROR    | app.utils:log_error:32 - TypeError(\"'age' is an invalid keyword argument for User\")\n", "record": {"elapsed": {"repr": "0:00:11.176632", "seconds": 11.176632}, "exception": null, "extra": {"user": "someone"}, "file": {"name": "utils.py", "path": "E:\\Pycharm\\learn-fast-api\\app\\utils.py"}, "function": "log_error", "level": {"icon": "\u274c", "name": "ERROR", "no": 40}, "line": 32, "message": "TypeError(\"'age' is an invalid keyword argument for User\")", "module": "utils", "name": "app.utils", "process": {"id": 7188, "name": "SpawnProcess-8"}, "thread": {"id": 15008, "name": "MainThread"}, "time": {"repr": "2021-06-23 23:28:27.855914+06:00", "timestamp": 1624469307.855914}}}
